akka-distributed-data/src/main/scala/akka/cluster/ddata/GSet.scala

```diff
@@ -4,15 +4,17 @@
 package akka.cluster.ddata
 
 object GSet {
-  private val _empty: GSet[Any] = new GSet(Set.empty)
+  private val _empty: GSet[Any] = new GSet(Set.empty)(None)
   def empty[A]: GSet[A] = _empty.asInstanceOf[GSet[A]]
   def apply(): GSet[Any] = _empty
+  def apply[A](set: Set[A]): GSet[A] = new GSet(set)(None)
+
   /**
    * Java API
    */
   def create[A](): GSet[A] = empty[A]
```

Comment:
I think we also need factory method for Java API, taking `java.util.Set[A]` param